eng/SourceBuild.props

```diff
@@ -24,6 +24,10 @@
   <Target Name="BuildBootstrap"
           DependsOnTargets="PrepareInnerSourceBuildRepoRoot"
           BeforeTargets="RunInnerSourceBuildCommand">
+          
+    <PropertyGroup>
+      <SourceBuildBootstrapTfmArg Condition="$(SourceBuildBootstrapTfm) != ''">--tfm $(SourceBuildBootstrapTfm)</SourceBuildBootstrapTfmArg>
```

Comment:
@mthalman Where does `SourceBuildBootstrapTfm` come from? is it something that is set up during VMR/SB builds?

Reply:
Yes, we'll configure the VMR build to set `SourceBuildBootstrapTfm` to the value of `NetCurrent` when invoking fsharp's build script.